src/operator/gitea.ts

```diff
@@ -468,44 +487,60 @@ async function setGiteaGroupMapping(podNamespace: string, podName: string) {
   }
 }
 
-async function setGiteaOIDCConfig() {
+async function setGiteaOIDCConfig(update = false) {
   if (!env.oidcClientId || !env.oidcClientSecret || !env.oidcEndpoint) return
   const podNamespace = 'gitea'
   const podName = 'gitea-0'
   const clientID = env.oidcClientId
   const clientSecret = env.oidcClientSecret
   const discoveryURL = `${env.oidcEndpoint}/.well-known/openid-configuration`
+  const teamNamespaceString = buildTeamString(env.teamNames)
 
   try {
     const execCommand = [
       'sh',
       '-c',
-      `AUTH_ID=$(gitea admin auth list --vertical-bars | grep -E "\\|otomi-idp\\s+\\|" | grep -iE "\\|OAuth2\\s+\\|" | awk -F " " '{print $1}' | tr -d '\\n') && gitea admin auth update-oauth --id "$AUTH_ID" --key "${clientID}" --secret "${clientSecret}" --auto-discover-url "${discoveryURL}"`,
+      `
+      AUTH_ID=$(gitea admin auth list --vertical-bars | grep -E "\\|otomi-idp\\s+\\|" | grep -iE "\\|OAuth2\\s+\\|" | awk -F " " '{print $1}' | tr -d '\\n')
+      if [ -z "$AUTH_ID" ]; then
+        echo "Gitea OIDC config not found. Adding OIDC config for otomi-idp."
+        gitea admin auth add-oauth --name "otomi-idp" --key "${clientID}" --secret "${clientSecret}" --auto-discover-url "${discoveryURL}" --provider "openidConnect" --admin-group "team-admin" --group-claim-name "groups" --group-team-map '${teamNamespaceString}'
+      elif ${update}; then
+        echo "Gitea OIDC config is different. Updating OIDC config for otomi-idp."
+        gitea admin auth update-oauth --id "$AUTH_ID" --key "${clientID}" --secret "${clientSecret}" --auto-discover-url "${discoveryURL}"
```

Comment:
why on update you do not use the `--group-claim-name "groups" --group-team-map '${teamNamespaceString}'` arguments

Reply:
thx